htdocs/theme/eldy/theme_vars.php

```diff
@@ -0,0 +1,286 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ *		\file       htdocs/theme/eldy/style.css.php
+ *		\brief      File for CSS style sheet Eldy
+ */
+
+//if (! defined('NOREQUIREUSER')) define('NOREQUIREUSER','1');	// Not disabled because need to load personalized language
+//if (! defined('NOREQUIREDB'))   define('NOREQUIREDB','1');	// Not disabled to increase speed. Language code is found on url.
+if (! defined('NOREQUIRESOC'))    define('NOREQUIRESOC', '1');
+//if (! defined('NOREQUIRETRAN')) define('NOREQUIRETRAN','1');	// Not disabled because need to do translations
+if (! defined('NOCSRFCHECK'))     define('NOCSRFCHECK', 1);
+if (! defined('NOTOKENRENEWAL'))  define('NOTOKENRENEWAL', 1);
+if (! defined('NOLOGIN'))         define('NOLOGIN', 1);          // File must be accessed by logon page so without login
+//if (! defined('NOREQUIREMENU'))   define('NOREQUIREMENU',1);  // We need top menu content
+if (! defined('NOREQUIREHTML'))   define('NOREQUIREHTML', 1);
+if (! defined('NOREQUIREAJAX'))   define('NOREQUIREAJAX', '1');
+
+
+// Colors
+$colorbackhmenu1='60,70,100';      // topmenu
+$colorbackvmenu1='248,248,248';      // vmenu
+$colortopbordertitle1='200,200,200';    // top border of title
+$colorbacktitle1='220,220,223';      // title of tables,list
+$colorbacktabcard1='255,255,255';  // card
+$colorbacktabactive='234,234,234';
+$colorbacklineimpair1='255,255,255';    // line impair
+$colorbacklineimpair2='255,255,255';    // line impair
+$colorbacklinepair1='250,250,250';    // line pair
+$colorbacklinepair2='250,250,250';    // line pair
+$colorbacklinepairhover='230,237,244';	// line hover
+$colorbacklinebreak='239,231,224';		// line break
+$colorbackbody='255,255,255';
+$colortexttitlenotab='100,60,20';
+$colortexttitle='0,0,0';
+$colortext='0,0,0';
+$colortextlink='0,0,100';
+$fontsize='0.86em';
+$fontsizesmaller='0.75em';
+$topMenuFontSize='1.1em';
+$toolTipBgColor='rgba(255, 255, 255, 0.96);';
+$toolTipFontColor='#333';
+
+// Badges colors
+$badgePrimary   ='#007bff';
+$badgeSecondary ='#6c757d';
+$badgeSuccess   ='#28a745';
+$badgeDanger    ='#dc3545';
+$badgeWarning   ='#ffc107';
+$badgeInfo      ='#17a2b8';
+$badgeDark      ='#343a40';
+$badgeLight     ='#f8f9fa';
+
+
+
+/* default color for status : After a quick check, somme status can have oposite function according to objects
+*  So this badges status uses default value according to theme eldy status img
+*  TODO: use color definition vars above for define badges color status X
+*/
+$badgeStatus0='#cbd3d3';
+$badgeStatus1='#bc9526';
+$badgeStatus2='#e6f0f0';
+$badgeStatus3='#bca52b';
+$badgeStatus4='#277d1e';
+$badgeStatus5='#cad2d2';
+$badgeStatus6='#cad2d2';
+$badgeStatus7='#baa32b';
+$badgeStatus8='#be3013';
+$badgeStatus9='#e7f0f0';
+
+
+
+
+
+
+if (defined('THEME_ONLY_CONSTANT')) return;
```

Comment:
The part from line 28 to 37 + 96 to 109 should be restore into main styles.css (new version line number)
They do not define variables but they start the output of data to requester.

Reply:
done